code/game/gamemodes/blob/powers.dm

```diff
@@ -25,7 +25,8 @@
 	if(blob_nodes.len)
 		var/list/nodes = list()
 		for(var/i = 1; i <= blob_nodes.len; i++)
-			nodes["Blob Node #[i]"] = blob_nodes[i]
+			var/obj/structure/blob/node/B = blob_nodes[i]
+			nodes["Blob Node #[i] ([get_location_name(B)])"] = B
```

Comment:
a new proc isn't needed you can just do `get_area(B)`

example: https://github.com/ParadiseSS13/Paradise/blob/de2b51ad965b05c0a82ea67180c10e6ae386f208/code/modules/events/traders.dm#L60

Reply:
`get_area(B)` works, but little differently than I would want, as I am really keen on trying to keep things consistent:

![GetAreaResults](https://user-images.githubusercontent.com/43862960/60464563-3a8a4380-9c4f-11e9-8607-2fbce566d146.png)

It's a small thing, and if maintainers rule out that it should be this way, I will change it.

Reply:
ah I see 

yeah why do some areas have `the `and some don't, weird